spec/resources/bare-text-nodes/style.css

```diff
@@ -0,0 +1,16 @@
+* {
+    color: black;
+    background-color: white;
+    font-family: monospace;
+    font-size: 20px;
```

Comment:
All of these settings are really to make it easier for Tesseract to OCR the text.